Include/object.h

```diff
@@ -790,15 +789,25 @@ static inline void _Py_ForgetReference(PyObject *op)
 {
     _Py_INC_TPFREES(op);
 }
+#endif /* !Py_TRACE_REFS */
+
 
-#ifdef Py_LIMITED_API
 PyAPI_FUNC(void) _Py_Dealloc(PyObject *);
```

Comment:
Maybe left it at line 771?

Reply:
I'm not sure that I understand your suggestion. Do you want me to put the definition back into the #ifdef Py_TRACE_REFS? I would like to make it very explicit that the function is now always defined and always defined, for any C define.

Reply:
Ah, I didn't notice that that list is into the #ifdef Py_TRACE_REFS. All correct then.